packages/logging/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@axiomhq/logging",
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./transports": {
+      "import": {
+        "types": "./dist/esm/transports/index.d.ts",
+        "default": "./dist/esm/transports/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/transports/index.d.cts",
+        "default": "./dist/cjs/transports/index.cjs"
+      }
+    }
+  },
+  "scripts": {
+    "build": "vite build"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@axiomhq/js": "^1.3.1"
```

Comment:
I think the version here needs to be `"workspace:*"`

Reply:
@gabrielelpidio this is still mentioning a fixed version instead of workspace version.